objectRecognitionSource/src/main/java/edu/ml/tensorflow/classifier/YOLOClassifier.java

```diff
@@ -19,20 +19,25 @@
  */
 public class YOLOClassifier {
     private final static float OVERLAP_THRESHOLD = 0.5f;
-    private final static double anchors[] = {1.08,1.19,  3.42,4.41,  6.63,11.38,  9.42,5.11,  16.62,10.52};
     private final static int SIZE = 13;
     private final static int MAX_RECOGNIZED_CLASSES = 24;
     private final static int MAX_RESULTS = 24;
-    private final static int NUMBER_OF_BOUNDING_BOX = 5;
+    public final static int NUMBER_OF_BOUNDING_BOX = 5;
+    
+    // Default anchor values used to properly label recognitions on original image
+    private static double anchors[] = {0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828};
```

Comment:
Maybe add a constant (private final...) called something like ANCHORS_FOR_PROVIDED_MODEL?  

Also, we should include a reference or description to the readme that outlines this a bit.  And maybe some of the links to the descriptive info that I found.  Also of interest, [object-detection](http://machinethink.net/blog/object-detection/) which talks about it... 

We should include a comment that says that ```anchor[]```  is NUMBER_OF_BOUNDING_BOX pairs of numbers, specifying the most common shapes of objects in the training data (within the YOLO 13x13 grid), and is used to construct the bounding boxes for detected objects.